lisa/schema.py

```diff
@@ -678,8 +680,9 @@ class Platform(TypedSchema, ExtendableSchemaMixin):
     admin_password: str = ""
     admin_private_key_file: str = ""
 
-    # True means not to delete an environment, even it's created by lisa
-    reserve_environment: bool = False
+    # no/False: means to delete the environment regardless case fail or pass
+    # yes/always/True: means to keep the environment regardless case fail or pass
+    reserve_environment: Optional[Union[str, bool]] = False
```

Comment:
reserve_environment: Optional[Union[ReserveEnvStatus, bool]] = False
this may be better?